backend/core/database.py

```diff
@@ -1304,11 +1314,11 @@ def _handle_bgp_withdrawals(self):
 
             try:
                 query = (
-                    "UPDATE bgp_updates SET handled=true, hijack_key=array_distinct(hijack_key || array[data.v1]) "
-                    "FROM (VALUES %s) AS data (v1, v2) WHERE bgp_updates.key=data.v2"
+                    "UPDATE bgp_updates SET handled=true, hijack_key=array_distinct(hijack_key || array[%s]) "
```

Comment:
I refactored the syntax to be compatible with execute_batch instead of values since values runs everything in parallel; good for performance, bad for correctness since there can be race conditions when updating the hijack keys for the same hijack (the most recent wins, which is erroneous).

Reply:
Update: I kept execute values for updates that do not have this dependency!